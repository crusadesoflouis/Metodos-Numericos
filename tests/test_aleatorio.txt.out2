0.85
0.229724
0.211282
0.229724
0.229724
0.0995469
