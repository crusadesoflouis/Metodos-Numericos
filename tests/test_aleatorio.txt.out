0.85
0.219302
0.0778443
0.202115
0.219302
0.281437
